mslearn-knowledge-mining.generated.sln

```diff
@@ -0,0 +1,46 @@
+﻿
```

Comment:
Generally we don't include sln files in these repos, and let users create their own if desired. we can remove this before merging, or I'll delete it after

Reply:
auto-generated. Now deleted